core/ipc/rpc/shell-commands/src/main/java/org/opennms/core/rpc/commands/MetaCommand.java

```diff
@@ -79,48 +84,53 @@ private void printScope(final Scope scope) {
         for (final Map.Entry<String, Set<ContextKey>> group : grouped.entrySet()) {
             System.out.printf("%s:\n", group.getKey());
             for (final ContextKey contextKey : group.getValue()) {
-                System.out.printf("  %s='%s'\n", contextKey.getKey(), scope.get(contextKey).get());
+                System.out.printf("  %s='%s'\n", contextKey.getKey(), scope.get(contextKey).isPresent() ? scope.get(contextKey).get() : "");
             }
         }
     }
 
     @Override
     public Object execute() throws Exception {
+
+        transactionOperations.execute(status -> {
         try {
-            final OnmsNode onmsNode = this.nodeDao.get(this.node);
-            if (onmsNode == null) {
-                System.out.printf("Cannot find node with ID/FS:FID=%s.\n", this.node);
+                final OnmsNode onmsNode = this.nodeDao.get(this.node);
+                if (onmsNode == null) {
+                    System.out.printf("Cannot find node with ID/FS:FID=%s.\n", this.node);
```

Comment:
This is misleading. `onmsNode` is the node object, where `this.node` is either a node id or a `FS:FID`. In the graph world I came up with the term `nodeRef` or `nodeReference` which can either be the id or `FS:FID`. I would suggest using that terminology here as well, as it makes things much more clear.